CHANGELOG.adoc

```diff
@@ -165,6 +165,7 @@ They still support returning an `java.time.Instant` object directly.
 - https://github.com/eclipse-sirius/sirius-web/issues/3450[#3450] [diagram] Add `calculateCustomNodeBorderNodePosition` method to position border nodes according to the real layout of the custom node
 - https://github.com/eclipse-sirius/sirius-web/issues/3460[#3460] [diagram] Use _arrange layout direction_ properties during edge handle position computing
 - https://github.com/eclipse-sirius/sirius-web/issues/3366[#3366] [gantt] Trigger the zoom on Ctrl+MouseWheel
+- https://github.com/eclipse-sirius/sirius-web/issues/3511[#3511] [diagram] Improve edge handle position for border nodes
```

Comment:
Are you sure that _handle_ is the appropriate term here? I thought _handle_ was referring to the four chevrons allowing to start linking an edge between two nodes.

Reply:
`Handle` is the term use by reactflow to define connection points for edges.
We use it for the creation (four chevrons) and to link existing edge to its node.

Reply:
Ok, thanks